packages/shared/lib/services/connection.service.ts

```diff
@@ -1543,9 +1545,6 @@ class ConnectionService {
             .where(function () {
                 this.where('_nango_sync_configs.active', true).orWhereNull('_nango_sync_configs.active');
             })
-            .where(function () {
-                this.where('_nango_sync_configs.enabled', true).orWhereNull('_nango_sync_configs.enabled');
-            })
```

Comment:
and we don't filter anymore the connections that don't have any enabled sync configs